app/javascript/article-form/components/ImageUploader.jsx

```diff
@@ -123,6 +123,48 @@ export const ImageUploader = () => {
     });
   }
 
+  function handleNativeMessage(e) {
+    let message = {};
+    try {
+      message = JSON.parse(e.target.value);
+    } catch (e) {
+      console.log(e); // eslint-disable-line no-console
```

Comment:
Refactor (blocking): Remove the `catch`/`console.log`. We have HoneyBader to catch client-side errors and this would swallow the error.

If there is something pro-active we can do in the catch such as asking the member to try again, we should consider doing that. Otherwise, let HoneyBadger notify us of the error.